ACE/tests/Dynamic_Priority_Test.cpp

```diff
@@ -47,9 +47,9 @@
 #include "ace/High_Res_Timer.h"
 #include "ace/Sched_Params.h"
 
-#if defined (VXWORKS)
+#if defined (VXWORKS) || defined (INTEGRITY)
 # include "ace/OS_NS_string.h"
```

Comment:
Why add INTEGRITY here?

Reply:
It's needed for the `ACE_OS::strlen()` function.